enginerpc/examples/engine-rpc-client.py

```diff
@@ -263,6 +277,17 @@ def check_arguments(client, args, flags):
         "same command line, you must only use one !\n\n" + colors.ENDC)
     exit(1)
 
+  if args.component == "broker":
+    client = gRPC_client("broker")
+  elif args.component == "engine":
+    client = gRPC_client("engine")
```

Comment:
not very beautiful this code :smile: 
gRPC_client(args.component) is also a possibility...